src/theory/logic_info.cpp

```diff
@@ -540,6 +540,12 @@ void LogicInfo::setLogicString(std::string logicString)
     IllegalArgument(logicString, err.str().c_str());
   }
 
+  if (d_theories[THEORY_SETS])
```

Comment:
This code should not be necessary.  Can you try deleting it?

Reply:
done